pyne/ionization_calculator.py

```diff
@@ -0,0 +1,598 @@
+#Begin the method to calculate ionionization energies in eV for various atoms and electronic states
+
+from pyne import nucname
+import numpy as np
+
+def function(element,charge):
+ 
+#initialize tuples 
```

Comment:
comments describing the functions should be in docstrings. 

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings